triplea_maps.yaml

```diff
@@ -705,6 +705,42 @@
     <br>Use forts, castles, siege weapons and the mighty dragons of war.
     <br>
     <br>
+- mapName: Iron War
+  mapCategory: GOOD
+  url: https://github.com/triplea-maps/iron_war/archive/master.zip
+  version: 12
+  img: https://raw.githubusercontent.com/triplea-maps/iron_war/master/description/MapThumbnail.png
+  description: |
+    <br><b><em>by Frostion</em></b>
+    <br>
+    <br>This map features 8 Axis and 12 Allied players.
+    <br>
+    <br>Play through WW2 as one or more Axis or Allied powers.
+    <br>Be free of restricting politics and non-aggression pacts.
+    <br>Purchase special units like SS, Colonial troops, Kamikaze plans etc.
+    <br>Keep track of Steel and Fuel income and availability on the map.
+    <br>Make use of the 25 different units this map has to offer.
+    <br>Enjoy custom map sounds and nation specific music.
+    <br>
+    <br>
+- mapName: Iron War: Europe
```

Comment:
Apologies, @FrostionAAA, but the code we normally run to validate this file is currently disabled (I'm working on getting it running again).  This line is malformed YAML because the colon (`:`) between "War" and "Europe" is not permitted in this context.  You'll need to quote the map name if you want to use a colon here.  For example:

```yaml
- mapName: "Iron War: Europe"
```

Reply:
Ohh, OK. I have just removed the : between the two words. Is it OK now?

Reply:
Yes, thanks.

I should have mentioned that the quotes will not be displayed in the UI, just in case that affected your decision.  They simply act as delimiters and are stripped by the YAML parser.

Let me know if you want to leave it as-is, and I'll go ahead and merge.  Otherwise, please push another commit if you want to go back to the version with the colon.

Reply:
OK. Thanks for the help :-) I have now changed it back, added the " and also changed some other names. Now it looks good.

Reply:
Hehe. OK. I hope this one works. I removed all the : and " now.